pyriemann/utils/mean.py

```diff
@@ -679,33 +674,29 @@ def mean_wasserstein(X, tol=10e-9, maxiter=50, init=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Geometric Radar Processing based on Frechet distance: Information
-        geometry versus Optimal Transport Theory
-        <https://ieeexplore.ieee.org/document/6042179>`_
-        F. Barbaresco. 12th International Radar Symposium (IRS), October 2011
+    .. [1] `Barycenter Estimation of Positive Semi-Definite Matrices with
+        Bures-Wasserstein Distance
+        <https://arxiv.org/abs/2302.14618>`_
+        J. Zheng, H. Huang, Y. Yi, Y. Li, S.-C. Lin.
     """
     n_matrices, n, _ = X.shape
     sample_weight = check_weights(sample_weight, n_matrices)
     if init is None:
         init = mean_euclid(X, sample_weight=sample_weight)
     else:
         init = check_init(init, n)
-    K = sqrtm(init)
-
-    crit = np.finfo(np.float64).max
+    K = init
```

Comment:
Can you keep `M` for the name of the mean?
```suggestion
    M = init
```